woocommerce/woocommerce
# Add customize store assembler hub logo

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #39715.

figma: Mrk6SERPZ4KrFHSjM0a8TK-fi-2631_256538

![Screenshot 2023-08-29 at 13 37 29](https://github.com/woocommerce/woocommerce/assets/4344253/23b3bd5b-8c2d-47e7-a113-f0dbf26dc002)

https://github.com/woocommerce/woocommerce/assets/4344253/bc5fe856-aac6-4631-b910-c62dc1f8ca4c




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use WP 6.3 
2. Install the `WooCommerce Beta Tester` plugin
3. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper and enable `customize-store` feature flag
4. Use a block theme with a logo block, such as `FSE Empire` theme, or add a logo block to your theme using site-editor.
5. Go to `WooCommerce > Home`
6. Click on `Customize your store` task 
7. Click on `Assembler Hub` button
8. Should see the Assembler Hub screen
9. Click on "Add your logo"
10. Confirm the UI matches the design
11. Upload a logo (You can also drag and drop to upload an image) and update the logo settings
12. Click on the save button
13. Open a new tab and go to the site frontend
14. Confirm the site logo is updated 
15. Confirm the site logo is clickable when "link logo to homepage" is enabled
16. Confirm the site icon is set when "use as site icon" is enabled.
17. Change the settings
18. Navigate back to the main page
19. Go to the logo page again
20. Changes should NOT be applied.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Implement customize store assembler hub - logo feature

#### Comment 

</details>
